### PR TITLE
Yamaha telnet fix

### PR DESCRIPTION
**Pull request by ktbyers**

### CODE REVIEW

**Comment by ktbyers** [comment]
FYI, some minor updates to your change here:

@Tachashi Tachashi

File: netmiko/yamaha/yamaha.py [hunk 1]
```diff
@@ -28,7 +28,7 @@ def exit_enable_mode(self, exit_command="exit"):
             time.sleep(1)
             output = self.read_channel()
             if "(Y/N)" in output:
-                self.write_channel("N")
+                self.write_channel(f"N{self.default_enter}")
```
Comment (ktbyers) [inline netmiko/yamaha/yamaha.py:31]: @Tachashi I moved the sending of the newline (\n) to here as I assumed the issue was that sending of the "N" required an <enter> at the end of it.

Let me know if this doesn't work for some reason i.e. you are still seeing issues with `exit_enable_mode`

**Comment by ktbyers** [comment]
I am going to go ahead and merge and if there are changes needed...we can just do a new PR.

**Comment by Tachashi** [comment]
@ktbyers Thanks. Sending "N" after the prompt "(Y/N)" succeeds without additional newline (\n). I opened new PR.

**Pull request merged**